templates/parachain/runtime/src/lib.rs

```diff
@@ -9,7 +9,7 @@ include!(concat!(env!("OUT_DIR"), "/wasm_binary.rs"));
 pub mod apis;
 #[cfg(feature = "runtime-benchmarks")]
 mod benchmarks;
-mod configs;
+pub mod configs;
```

Comment:
This is useful when `node` needs access some parameters, although the parachain node doesn't reply on it, I still sync the change from solochain template